src/cljs/athens/devcards/athena.cljs

```diff
@@ -178,28 +246,27 @@
                         (swap! *cache assoc query result)
                         (reset! *match [query result])))))]
     (when @athena?
-      [:div +container
-       [:div {:style {:box-shadow "inset 0px -1px 0px rgba(0, 0, 0, 0.1)"}}
-        [:input (with-attributes +athena-input
-                  {:type        "search"
-                   :placeholder "Find or Create Page",
-                   :on-change   handler})]]
+      [:div (use-style container-style)
+       [:input (use-style athena-input-style
+                          {:type        "search"
+                           :auto-focus  true
+                           :placeholder "Find or Create Page"
+                           :on-change   handler})]
        [recent]
        [(fn []
           (let [[query {:keys [pages blocks] :as result}] @*match]
             (when result
-              [:div (with-styles +query)
+              [:div (use-style results-list-style)
                (for [[i x] (map-indexed list (take 40 (concat (take 20 pages) blocks)))]
                  (let [parent (:block/parent x)
                        page-title (or (:node/title parent) (:node/title x))
                        block-uid (or (:block/uid parent) (:block/uid x))
                        block-string (:block/string x)]
-                   [:div (with-attributes {:class "athena-result" :key i :on-click #(navigate-page block-uid)})
-                    [:div
-                     [:h4 (highlight-match query page-title)]
-                     (when block-string
-                       [:span (highlight-match query block-string)])]
-                    [:h4 (with-styles {:margin-left "auto"}) [:> mui-icons/ChevronRight]]]))])))]])))
+                   [:div (use-style result-style {:key i :on-click #(navigate-page block-uid)})
+                    [:h4.title (use-sub-style result-style :title) (highlight-match query page-title)]
+                    (when block-string
+                      [:span.preview (use-sub-style result-style :preview) (highlight-match query block-string)])
```

Comment:
The UX of search results is a rabbithole, so these are just comments for the future. You've probably already thought of all this.

- I'd personally want orange highlight, not just heavier font weight, for more contrast.
- Sometimes the block-string is matched pretty far within the block, so we don't see the closest context. We only see the start of the block and the title of the page it's in.

Reply:
@tangjeff0 agreed.
- The highlight was supposed to be black according to the mockups, so a little bit darker and clearer, but I wasn't able to get them to change on row hover until just now. That's fixed in the cleanup PR. Open to discussing more contrast.
- Until we're able to trim the string, this is a tradeoff between sometimes clipping out the term and sometimes just seeing enormous block previews. Clipping to 1 line is probably overzealous, so I'll show the whole result until a decision is made or the trimming feature is added.